source/codegen/metadata/nifgen/functions.py

```diff
@@ -726,26 +726,6 @@
         ],
         'returns':'ViStatus'
     },
```

Comment:
@Akshata440  @kruthi-NI  is there a way we could update Kevin's script so as to not migrate metadata for obsolete functions? This comment shouldn't hold off this review, but I wanted to us to evaluate this.

Reply:
Some drivers will have Obsolete comment for the section of obsolete APIs in the actual header itself. But some will have those APIs in a separate header like <driver_name>.obsolete.h. 
So if we can parse the main header and filter all the public API names(by excluding APIs if any in Obsolete section), then we might try to use that list. But I don't know how accurate it can be and how much effort it might be

Reply:
Ok. I was thinking more of if there is some information in the metadata itself that we could use to distinguish between obsolete and non-obsolete APIs, we could've used that. If it is more than that, we don't need to do it now and do it later if we need to use the script again.

Reply:
We use Kevin's script to convert the functions.hapigen into the format we require for grpc. But in functions.hapigen I dont see any obsolete tags for the obsoleted APIs
![image](https://user-images.githubusercontent.com/47028785/120785444-c2b7a880-c54a-11eb-85f8-6fbed640f021.png)
So I dont think we can update script to leave out these APIs

Reply:
We can leave it out then.